### ISSUE TITLE
Unit values are not checked properly

### ISSUE THREAD

**Issue by NikolayFrantsev** [SimenB/stylint#230]
➜ cat ok.styl 

```
@keyframes preloader
  0%
    background-position:0%

  100%
    background-position:100%
```

➜ cat fail.styl 

```
@keyframes preloader
  0%
    background-position:100%

  100%
    background-position:0%
```

➜ stylint ok.styl 
➜ stylint fail.styl 

```
Warning: 0 is preferred. Unit value is unnecessary
File: fail.styl
Line: 6: background-position:0%


Stylint: 0 Errors.
Stylint: 1 Warnings.
```

**Comment by rossPatton** [comment]
thanks, this error shouldn't trigger inside @keyframes, i'll take a look

**Comment by Shutnik** [comment]
error is not triggered on keyframe times, but I'm not sure if it's good to ignore errors for properties inside keyframe times
